PROJECT

```diff
@@ -1,6 +1,6 @@
 domain: openshift.io
 layout: go.kubebuilder.io/v2
-repo: github.com/RiRa12621/openshift-route-monitor-operator
+repo: github.com/openshift-route-monitor-operator
```

Comment:
isn't it `openshift/route-monitor-operator` ?

Reply:
yeah, I did the fix like 5 times and I missed this one, will fix